### ISSUE TITLE
add a little visual spot next to a topic while hovered over, that when grabbed starts synapse creation

### ISSUE THREAD

**Issue by Connoropolous** [metamaps/metamaps#960]

**Comment by Connoropolous** [comment]
further idea for this: 
make it a tiny circle that appears around the periphery of the topic you are hovering over, in the position that is closest to the position of your mouse. so as you move your mouse around the topic (or initially over it) the little synapse creation circle is always the closest to your mouse that it could be. 

In addition, this little visual circle could appear around all topics, if you were on a touch screen, to solve for the synapse creation on touch screen problem (where you don't have right click or alt buttons)

**Comment by Connoropolous** [comment]
I think implementing this will really help new, and existing users, get started creating synapses

**Comment by Bortseb** [comment]
it also makes mouse-only interaction on MAC possible right? I only ever use right-drag to be honest.

**Comment by Connoropolous** [comment]
Yes, if you were on a Mac, and using a mouse, this would make it easier /nicer than having to hold down alt

**Comment by Bortseb** [comment]
100BD/C == 100x0.5x0.75/10 = 3.8

I estimated that this might take 10 hours... is this accurate @Connoropolous ?

**Comment by Connoropolous** [comment]
depends who does it :p 

to add a little more thought to it here:
there's a design component, what does this little bubble thing look/feel like? 
do we need an image asset, or can the clickable thing be drawn onto the html5 canvas?

hm, yeah it seems like 10 is a useful/fine number to use here

**Comment by Bortseb** [comment]
btw... I really like this idea... and also the visual effect of it following your mouse so that you can more easily click on it.

**Comment by Connoropolous** [comment]
I actually think it would affect most users (perhaps future/new users) a lot, because creating a synapse hasn't been highly intuitive up to this point. I am actually pretty energized to do it.

**Comment by Bortseb** [comment]
That is true... my low breadth score is likely biased to myself/existing users.... Where we already know how to create synapses other ways and might not bother with trying to specifically click on the new "button"

But you are right... new users, who either wouldn't think to right-drag or can't, would likely find playing with the button more intuitive...

**Comment by Bortseb** [comment]
100BD/C = 100x0.8x0.75/10 = 6

**Comment by Bortseb** [comment]
I think the "icon" could even be a little vector/arrow symbol always pointing out from the centre of the metacode to make it even more obvious that it has to do with connecting "arrows"/lines between things.

**Comment by Connoropolous** [comment]
I like that

**Comment by Connoropolous** [comment]
I think this should be fairly highly prioritized
